### PR TITLE
Studio: User profile pill

### PR DESCRIPTION
**Pull request by achen5671**
## Summary
https://tsu.atlassian.net/browse/IR-2520
## References
closes #_insert number here_

## QA Steps

Screenshot 2024-10-01 at 1 05 46 PM
Screenshot 2024-10-01 at 1 05 49 PM

### CODE REVIEW

**Comment by aditya-mitra** [comment]
@achen5671 screenshots for these too
(helps point out the changes faster)

File: packages/ui/src/components/tailwind/ContextMenu/index.tsx
Comment (aditya-mitra) [inline packages/ui/src/components/tailwind/ContextMenu/index.tsx:1]: revert the change on this file too

File: packages/editor/src/components/toolbar/Toolbar.tsx [hunk 1]
```diff
@@ -143,7 +145,8 @@ const toolbarMenu = generateToolbarMenu()
 
 export default function Toolbar() {
   const { t } = useTranslation()
-  const anchorEvent = useHookstate<null | React.MouseEvent<HTMLElement>>(null)
+  const toolbarAnchorEvent = useHookstate<null | React.MouseEvent<HTMLElement>>(null)
+  const profileAnchorEvent = useHookstate<null | React.MouseEvent<HTMLElement>>(null)
```
Comment (aditya-mitra) [inline packages/editor/src/components/toolbar/Toolbar.tsx]: can we revert these?
also can you remove the unused variables

**Review by aditya-mitra — DISMISSED** [review]
almost done.
can you implement these 3 states that are being shown in the [ir design library figma](https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi/iR-Engine-Design-Library-File?node-id=2839-16186&node-type=frame&t=3utcM0pxpolUGylP-0)?

image

**Comment by achen5671** [comment]
> almost done. can you implement these 3 states that are being shown in the [ir design library figma](https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi/iR-Engine-Design-Library-File?node-id=2839-16186&node-type=frame&t=3utcM0pxpolUGylP-0)?
> 
> image

Im not too sure what the difference is between the default and hover state. Ill add the focus state though

**Comment by achen5671** [comment]
> > almost done. can you implement these 3 states that are being shown in the [ir design library figma](https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi/iR-Engine-Design-Library-File?node-id=2839-16186&node-type=frame&t=3utcM0pxpolUGylP-0)?
> > image
> 
> Im not too sure what the difference is between the default and hover state. Ill add the focus state though

Screenshot 2024-10-04 at 9 29 36 AM

**Review by JT00y — APPROVED** [review]
looks good, popover modal shows correct

**Pull request merged**